backend/Dockerfile

```diff
@@ -4,14 +4,28 @@
 ARG BASE_IMAGE=python:3.11-bullseye
 FROM $BASE_IMAGE
 
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+ENV SHELL=/bin/bash
+
 RUN apt-get update && \
     apt-get upgrade --yes && \
     apt-get install --yes \
     ca-certificates \
     unzip \
     libpq-dev \
+    curl \
     && rm -rf /var/lib/apt/lists/*
 
+# Install kubectl if not available in base image
+# kubectl is needed for the list files endpoint
+RUN dpkg -s kubectl || kubectl_installed="no"; \
+    if [[ "$kubectl_installed" == "no" ]]; \
+    then \
+    curl -fsSLo /usr/share/keyrings/kubectl.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg; \
+    echo "deb [signed-by=/usr/share/keyrings/kubectl.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list; \
+    apt-get update && apt-get install -y kubectl && rm -rf /var/lib/apt/lists/*; \
+    fi
+
```

Comment:
Why not "just" install it?

Reply:
In our DB environment pipelines, we don't have internet access. The install command would fail (`https://apt.kubernetes.io/` is not reachable). But it is already installed in our Base image, we can skip the installation.